1.8/usage/containerizers.md

```diff
@@ -0,0 +1,56 @@
+---
+post_title: Choosing a Containerizer
+nav_title: Containerizers
+menu_order: 3.5
+---
+
+DC/OS supports both [the default Mesos containerizer and the Docker containerizer](http://mesos.apache.org/documentation/latest/containerizer/).
+
+# Mesos Containerizer
+
+While other containerizers play well with DC/OS, the Mesos containerizer does not depend upon other container technologies and can therefore take advantage of more Mesos features.
```

Comment:
The [Mesos documentation](http://mesos.apache.org/documentation/latest/container-image/) only talks about why the unification makes sense for Mesos. But at the moment, there's a feature disparity between the containerizers. E.g. GPU support is only available for the Mesos containerizer. Eventually, support for the Docker containerizer will be implemented as well though.